.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: Test CI
+on: push
+jobs:
+  Test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        erlang: ['21.1', '20.3']
+    container:
+      image: erlang:${{matrix.erlang}}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/cache@v3
+        env:
+          cache-name: rebar3
+        with:
+          path: |
+           ~/.cache/rebar3
+           _build
+          key: ci-${{runner.os}}-${{env.cache-name}}-erlang_${{matrix.erlang}}-rebar3-${{hashFiles('rebar.lock')}}
+          restore-keys: |
+            ci-${{runner.os}}-${{env.cache-name}}-erlang_${{matrix.erlang}}-rebar3
+            ci-${{runner.os}}-${{env.cache-name}}-erlang_${{matrix.erlang}}
+      - name: elvis
+        run: make elvis
```

Comment:
Seperated `make test` into individual `make` steps to have an easier way to identify which step failed if a failure occurs
<img width="1064" alt="image" src="https://github.com/adgear/consent-string/assets/71024958/c8304660-082f-49b1-918d-ddbeb626f1e9">